quickstarts/gradle/thorntail/build.gradle

```diff
@@ -0,0 +1,77 @@
+buildscript {
+  String thorntailVersion = System.getProperty('thorntailVersion') ?: VERSION_THORNTAIL
+
+  repositories {
+    mavenLocal()
+    mavenCentral()
+    maven {
+      url "https://oss.sonatype.org/content/repositories/snapshots/"
+    }
+  }
+
+  dependencies {
+    classpath "io.thorntail:thorntail-gradle-plugin:$thorntailVersion"
+    classpath 'org.eclipse.jkube.kubernetes:org.eclipse.jkube.kubernetes.gradle.plugin:1.8.0-SNAPSHOT'
+    classpath 'org.eclipse.jkube.openshift:org.eclipse.jkube.openshift.gradle.plugin:1.8.0-SNAPSHOT'
```

Comment:
This requires changes here:
https://github.com/eclipse/jkube/blob/627832bf33b954b4f7c2c5bc32e3da0a6e20116a/scripts/quickstarts.sh#L30-L39